.github/CODEOWNERS

```diff
@@ -296,7 +296,7 @@
 
 # > Service: Cosmos
 # ServiceLabel: %Cosmos %Service Attention
-#/<NotInRepo>/            @Wmengmsft @MehaKaushik @shurd @anfeldma-ms @zfoster @kushagraThapar @southpolesteve
+/sdk/cosmosdb/          @Wmengmsft @MehaKaushik @shurd @anfeldma-ms @zfoster @kushagraThapar @southpolesteve
```

Comment:
This one might just be for the resource manager for Cosmos... not for all of Cosmos... We should ask